app/controllers/frontend_log_controller.rb

```diff
@@ -2,6 +2,7 @@ class FrontendLogController < ApplicationController
   respond_to :json
 
   skip_before_action :verify_authenticity_token
+  before_action :override_ahoy_user
```

Comment:
I figured an opt in was much safer than putting a before_action in app controller...